project/assets/evaluations.less

```diff
@@ -0,0 +1,81 @@
+
+
+body{
+  background-color: rgb(82, 86, 89);
+  margin-top: 50px;
+  margin-left: 0;
+}
+
+.navigation-bar {
+  height: 80px;
+  background-color: white;
+  border-bottom: 1px solid darkgrey;
+
+
+  .navigation-bar-svlogo {
+    width: 200px;
+    height: 40px;
+    margin-left: 60px;
+    margin-top: 10px;
+  }
+
+  .navigation-bar-active {
+    border-bottom: 6px solid DodgerBlue;
+    padding:18px;
+  }
+
+  .navigation-bar-logout {
+    margin-left: 830px;
+  }
+
+}
+
+
+
+.content {
+  background-color: white;
+  width: 1895px;
+  height: 800px;
+
+
+  table {
+    border-collapse: collapse;
+    width: 1200px;
+    height: 300px;
+    position:absolute;
+    margin-left: 400px;
+    margin-top: 180px;
+    text-align: left;
+    font-size:20px;
+  }
+
+  th {
+    background-color: #0080ff;
+    color: white;
+    font-size: 25px;
+    font-weight: normal;
+  }
+
+  td {
+    color: #4d4d4d;
+  }
+
+  tr {
+    border-bottom: 1px solid lightgrey;
+  }
+
+}
+
+.page-footer {
+  background-color: lightgrey;
+  margin-top: -20px;
+  width: 1897px;
+  height: 80px;
+  color: grey;
+  font-size: 20px;
+
+  .page-footer-text {
+    margin-left: 1400px;
```

Comment:
If only we could align text inside a container without using a margin... Do you know how we could achieve this?